ChangeLog.md

```diff
@@ -1,5 +1,171 @@
 # Changelog for haskell-language-server
 
+## 0.9.0
+
+This release includes lot of refactorings and bug fixes over existing features, hlint and eval plugins among others.
+The list of contributors keeps growing healthy, many thanks to you all!
```

Comment:
```suggestion
The list of contributors keeps growing, many thanks to you all!
```

Reply:
Perhaps

"continues to show healthy growth"?